src/ripple/rpc/impl/KeypairForSignature.h

```diff
@@ -33,6 +34,7 @@ struct KeyPair
 };
 
 KeyPair keypairForSignature (Json::Value const& params, Json::Value& error);
```

Comment:
I'm strongly against proliferation of structs designed to be "better" than `std::pair`. Rationale: `pair` is very well understood and while the names `first` and `last` are a deficiency in terms of being self-documenting, the benefits of having a "complete" type outweigh the cost. Besides, you can always write:
```
RippleAddress pk;
RippleAddress sk;
std::tie(pk, sk) = generateKeyPairFromSeed;
```

Something which is _not possible_ with a custom struct.